GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -70,6 +71,8 @@ public partial class FormPush : GitModuleForm
             new TranslationString("Force push may overwrite changes since your last fetch. Do you want to use the safer force with lease instead?");
         private readonly TranslationString _forceWithLeaseTooltips =
             new TranslationString("Force with lease is a safer way to force push. It ensures you only overwrite work that you have seen in your local repository");
+
+        private readonly TranslationString _errorCaption = new TranslationString("Error");
```

Comment:
Is it because that already exist? or should we use a more meaning full text ('No remotes configured'?).

Reply:
This a caption for a message box

Reply:
yes, I see that. But I'm not always pleased when I see caption not meaningful and not helping at all.
I just would like to know if you did it by convenience because the translated string already exist? Or you introduced it?
It's not a big deal...
But someone it helps the user (to quickly understand because he is used to) when the popup is displayed often. But here, it should quite never happen.

Reply:
It is a valid point
I found [Windows design guidelines](https://msdn.microsoft.com/en-us/library/windows/desktop/dn742471.aspx), I'll adjust the title accordingly